test/ScreeningAverageRisk/cases/GradeDErrorMissingCytologyResultEmptyConclusionCode.yml

```diff
@@ -0,0 +1,26 @@
+---
+name: Grade D Error Missing Cytology Result Empty Conclusion Code
```

Comment:
How come we only generate an error for the grade d exclusions @mickohanlon23 ?

Reply:
I believe this was answered in [the above comment](https://github.com/ccsm-cds-tools/ccsm-cds-with-tests/pull/4#discussion_r753897699), but essentially we still need to add Error cases for Diagnostic Reports in the Dashboard, which should ultimately cover the rest of the DiagnosticReport queries that use expressions from the Dashboard.